src/index.html

```diff
@@ -11,7 +11,195 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      class="form"
+    >
+      <fieldset class="form_info form">
+        <legend>Personal information</legend>
+        <div class="info_surname form-block">
+          <label>
+            Surname:
+            <input
+              type="text"
+              name="surname"
+              id="surname"
+              autocomplete="off"
+            >
+          </label>
+        </div>
+        <div class="info_name form-block">
+          <label>
+            Name:
+            <input
+              type="text"
+              name="name"
+              id="name"
+              autocomplete="off"
+            >
+          </label>
+        </div>
+        <div class="info_age form-block">
+          <label>
+            How old are You?
+            <input
+              type="number"
+              name="age"
+              id="age"
+              max="100"
+              min="1"
+              value="12"
+            >
+          </label>
```

Comment:
There are many unused attributes here, better to avoid id. Specify only that attributes that you need

For example Id is not used for styles or labels, so it can be removed
Same with classes `info_age`, `info_surname`, etc.